app/controllers/challenges_controller.rb

```diff
@@ -169,6 +169,8 @@ def update
       badge_params[:nsfw_level] = allowed_challenge_params[:nsfw_level] || @challenge.nsfw_level
 
       if !failure && @challenge.update(allowed_challenge_params) && @badge.update(badge_params) && @badge_map.update(allowed_badge_map_params)
+        Participations.where(challenge_id: @challenge.id).update_all(start_date: new_challenge.start_date)
+        Participations.where(challenge_id: @challenge.id).update_all(end_date: new_challenge.end_date)
```

Comment:
its newChallenge, not new_challenge
that said, i would suggest fixing this by correcting "newChallenge" to "new_challenge", snake_case is ruby standard, the camelCase is javascript standard

Reply:
well done!